### ISSUE TITLE
Add 'Accessibility Remediation Request' form to LDR footer

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#939]
The Libraries Accessibility Team has created a form for users to request remediated versions of resources in the LDR (i.e. a screenreader-optimized PDF).  

Form URL = https://forms.gle/T8sTupUsxQ8T6kwk6

This link should be added to the footer links that is visible at the bottom of all pages of the LDR. 
The text can read 'Accessibility Remediation Request'. 

<img width="1660" alt="Screen Shot 2022-05-26 at 3 47 34 PM" src="https://user-images.githubusercontent.com/38009796/170574062-39ee0ebf-5cd0-4cbc-9c91-ba7ccfbc9a6c.png">

**Comment by rococodogs** [comment]
closed with #945
